### PR TITLE
fix(sdcm/cluster_k8s): freeze helm version

### PR DESCRIPTION
**Pull request by dkropachev**
https://trello.com/c/HpgLCRp7/2369-fix-helm-33-issue

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Review by enaydanov — APPROVED** [review]
LGTM

File: sdcm/cluster_k8s.py [hunk 1]
```diff
@@ -107,9 +107,10 @@ def setup_minikube_ubuntu18(node: cluster.BaseNode, kubectl_version: str, miniku
             chmod +x /usr/local/bin/minikube
 
             # Install Helm 3.
-            snap install helm --classic
+            snap install helm --channel=3.2/stable --classic
             ln -s /snap/bin/helm /usr/local/bin/helm
-        """)
+            chmod +x /usr/local/bin/helm
+            """)
```
Comment (enaydanov) [inline sdcm/cluster_k8s.py:113]: Don't think we need these changes.

**Review by enaydanov — CHANGES_REQUESTED** [review]

File: sdcm/cluster_k8s.py (hunk 1 above)
Comment (bentsi) [reply sdcm/cluster_k8s.py:113]: @dkropachev why did you add the `chmod` line?

**Review by enaydanov — APPROVED** [review]
LGTM

**Review by bentsi — APPROVED** [review]

**Pull request merged**